stdlib/private/StdlibCollectionUnittest/CheckRangeReplaceableCollectionType.swift

```diff
@@ -862,6 +862,22 @@ self.test("\(testNamePrefix).removeSubrange()/semantics") {
   }
 }
 
+self.test("\(testNamePrefix).replaceSubrange()/closedRange/semantics") {
+  for test in removeRangeTests {
```

Comment:
Yes. Since we use the filter-collection-of-tests pattern elsewhere we should use it here too.